app/views/play/menu/GuideView.coffee

```diff
@@ -122,6 +122,7 @@ module.exports = class LevelGuideView extends CocoView
     tag.height = @helpVideoHeight
     tag.width = @helpVideoWidth
     tag.frameborder = '0'
```

Comment:
By the way, here's another issue: although the attribute is called `frameborder`, the corresponding DOM element property is named `frameBorder`.

This issue is not visible in Chrome, as Chrome does not display a border around `iframe`s by default. The border is visible in other browsers such as Firefox, though.

I believe it may be better to fix this using CSS (e.g. `iframe { border: none; }`) rather than DOM properties.